ts/dx.all.d.ts

```diff
@@ -1522,273 +1522,6 @@ declare module DevExpress.exporter {
         value?: any;
     }
 }
-declare module DevExpress.framework {
```

Comment:
@IlyaKhD Is it ok to do this in the context of this PR?

Reply:
Yes, a PR that updates declarations (doc-comments) is supposed to have corresponding TS updates as well